tests/e2e/hooks.js

```diff
@@ -0,0 +1,32 @@
+const { Before, BeforeAll, AfterAll, After, setDefaultTimeout } = require('@cucumber/cucumber')
+const { deleteFile } = require('./utils/helpers')
+const config = require('./config')
+const { chromium } = require('@playwright/test')
+// default timeout (in milliseconds)
```

Comment:
good to have separation between imorts and others
```suggestion
const { chromium } = require('@playwright/test')

// default timeout (in milliseconds)
```